pkg/solana/logpoller/worker/worker.go

```diff
@@ -21,11 +21,11 @@ var (
 
 const (
 	// DefaultMaxRetryCount is the number of times a job will be retried before being dropped.
-	DefaultMaxRetryCount = 6
+	DefaultMaxRetryCount = 18 // give up after ~ 14 hours
 	// DefaultNotifyRetryDepth is the retry queue depth at which the worker group will log a warning.
-	DefaultNotifyRetryDepth = 200
+	DefaultNotifyRetryDepth = 2000
```

Comment:
Increasing these to reduce noise--even for our manual e2e testing in staging it's normal for it to get up to at least 1600